src/main/resources/dbreplica/migration/V1.0.33__reportable_task_plsql.sql

```diff
@@ -0,0 +1,324 @@
+l_rt_due_date_to_completed_diff_days INTEGER;
+l_rt_completed_date                  DATE;
+l_rt_completed_date_time             TIMESTAMP;
+l_rt_first_assigned_date             DATE;
+l_rt_first_assigned_date_time        TIMESTAMP;
+l_rt_number_of_reassignments         INTEGER;
+l_rt_due_date                        DATE;
+l_rt_last_updated_date               DATE;
+l_rt_wait_time                       INTERVAL;
+l_rt_handling_time                   INTERVAL;
+l_rt_processing_time                 INTERVAL;
+l_rt_due_date_to_completed_diff_time INTERVAL;
+
+task_history_cursor CURSOR FOR
+SELECT task_id,task_name,task_type,due_date_time,state,task_system,security_classification,title,description,notes,
+       major_priority,minor_priority,assignee,auto_assigned,execution_type_code,work_type,role_category,has_warnings,
+       assignment_expiry,case_id,case_type_id,case_category,case_name,jurisdiction,region,region_name,location,
+       location_name,business_context,termination_reason,created,updated_by,updated,update_action,additional_properties,
+       reconfigure_request_time,next_hearing_id,next_hearing_date,priority_date,last_reconfiguration_time
+FROM cft_task_db.task_history
+WHERE task_id = l_task_id order by updated;
+
+begin
+
+select not exists(select 1 from cft_task_db.reportable_task where reportable_task.task_id = l_task_id)
+into l_new_task;
+
+OPEN task_history_cursor;
+
+LOOP
+FETCH NEXT FROM task_history_cursor INTO
+    l_task_id,l_task_name,l_task_type,l_due_date_time,l_state,l_task_system,l_security_classification,l_title,l_description,l_notes,
+    l_major_priority,l_minor_priority,l_assignee,l_auto_assigned,l_execution_type_code,l_work_type,l_role_category,l_has_warnings,
+    l_assignment_expiry,l_case_id,l_case_type_id,l_case_category,l_case_name,l_jurisdiction,l_region,l_region_name,l_location,
+    l_location_name,l_business_context,l_termination_reason,l_created,l_updated_by,l_updated,l_update_action,l_additional_properties,
+    l_reconfigure_request_time,l_next_hearing_id,l_next_hearing_date,l_priority_date,l_last_reconfiguration_time;
+
+    -- Exit the loop if no more rows are available
+    EXIT WHEN NOT FOUND;
+
+    l_role_category_label =
+        case l_role_category IS NOT NULL
+            when (l_role_category='LEGAL_OPERATIONS')  then 'Legal Operations'
+            when (l_role_category='CTSC')  then 'CTSC'
+            when (l_role_category='JUDICIAL')  then 'Judicial'
+            when (l_role_category='ADMIN')  then 'Admin'
+            else l_role_category
+        end;
+
+    l_jurisdiction_label =
+        case l_jurisdiction IS NOT NULL
+            when (l_jurisdiction='PRIVATELAW')  then 'Private Law'
+            when (l_jurisdiction='CIVIL')  then 'Civil'
+            when (l_jurisdiction='IA')  then 'Immigration and Asylum'
+            when (l_jurisdiction='PUBLICLAW')  then 'Public Law'
+            when (l_jurisdiction='EMPLOYMENT')  then 'Employment'
```

Comment:
As discussed please check that SSCS are happy for their label to be `SSCS` rather than something else as we can add their mapping here too.